### PR TITLE
{phys}[foss/2019a] BerkeleyGW v2.1.0, ELPA v2018.11.001 (WIP)

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/b/BerkeleyGW/BerkeleyGW-2.1.0_check.patch [hunk 1]
```diff
@@ -0,0 +1,38 @@
+--- Makefile.orig	2019-07-23 17:40:16.489733901 +0800
```
Comment (Micket) [inline easybuild/easyconfigs/b/BerkeleyGW/BerkeleyGW-2.1.0_check.patch]: Description and author

File: easybuild/easyconfigs/e/ELPA/ELPA-2018.11.001-foss-2019a.eb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+#
+# Authors::   Inge Gutheil <i.gutheil@fz-juelich.de>, Alan O'Cais <a.ocais@fz-juelich.de>
+# License::   MIT/GPL
+#
+##
+
+name = 'ELPA'
+version = '2018.11.001'
+
+homepage = 'http://elpa.rzg.mpg.de'
```
Comment (Micket) [inline easybuild/easyconfigs/e/ELPA/ELPA-2018.11.001-foss-2019a.eb]: https

File: easybuild/easyconfigs/e/ELPA/ELPA-2018.11.001-foss-2019a.eb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+description = """Eigenvalue SoLvers for Petaflop-Applications ."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+toolchainopts = {'openmp': True, 'usempi': True}
+
+source_urls = ['http://elpa.mpcdf.mpg.de/html/Releases/%(version)s/']
```
Comment (Micket) [inline easybuild/easyconfigs/e/ELPA/ELPA-2018.11.001-foss-2019a.eb]: https

File: easybuild/easyconfigs/b/BerkeleyGW/BerkeleyGW-2.1.0-foss-2019a.eb [hunk 4]
```diff
@@ -0,0 +1,63 @@
+easyblock = 'ConfigureMake'
+
+name = 'BerkeleyGW'
+version = '2.1.0'
+
+homepage = 'http://www.berkeleygw.org'
```
Comment (Micket) [inline easybuild/easyconfigs/b/BerkeleyGW/BerkeleyGW-2.1.0-foss-2019a.eb]: https

File: easybuild/easyconfigs/b/BerkeleyGW/BerkeleyGW-2.1.0-intel-2019a.eb [hunk 5]
```diff
@@ -0,0 +1,60 @@
+easyblock = 'ConfigureMake'
+
+name = 'BerkeleyGW'
+version = '2.1.0'
+
+homepage = 'http://www.berkeleygw.org'
```
Comment (Micket) [inline easybuild/easyconfigs/b/BerkeleyGW/BerkeleyGW-2.1.0-intel-2019a.eb]: https

**Comment by migueldiascosta** [comment]
this is still WIP because one test is failing when using the intel compilers (I can make it pass but it's weird...)

**Comment by migueldiascosta** [comment]
closed in favour of #9855

**Pull request closed without merging**